credential-issue-and-verification-api-user-guide.md

```diff
@@ -25,7 +25,7 @@ In your browser you can start three tabs to execute the APIs using swagger.
 
 # Schema definition by a legal entity
 On [Test Center Agent](agent1.swagger.localhost), you can execute the schema definiton API to register the schema in the ledger. Ideally this is defined by a legal entity or a standardisation body.
-
+To define schema first you have to create a public DID. To create public DID follow step 1 of [Credenial issuance by the issuer (Test Center)](# Credenial issuance by the issuer (Test Center))
```

Comment:
@Selbin Can you fix the link to `#credenial-issuance-by-the-issuer-test-center`

Reply:
Noticed that the link to local agent swagger is broken. Please check others as well and fix them.

Reply:
ok

Reply:
fixed those issues